test/src/main/java/org/apache/accumulo/test/ExportTableCommandWitlMultipleVolumes.java

```diff
@@ -0,0 +1,156 @@
+    baseDir2Array[2] = baseDir2Array[2] + "2";
+    String secondVolume = baseDir2Array[2];
+
+    // make second volume base directory
+    for (String element : baseDir2Array) {
+      baseDir2Str = baseDir2Str + "/" + element;
+    }
+    File baseDir2 = new File(baseDir2Str);
+
+    File v1f = new File(baseDir, "volumes/v1");
+    File v2f = new File(baseDir2, "volumes/v2");
+
+    v1 = new Path("file://" + v1f.getAbsolutePath());
+    v2 = new Path("file://" + v2f.getAbsolutePath());
+
+    // Run MAC on two locations in the local file system
+    cfg.setProperty(Property.INSTANCE_VOLUMES, v1 + "," + v2);
+
+    // use raw local file system so walogs sync and flush will work
+    hadoopCoreSite.set("fs.file.impl", RawLocalFileSystem.class.getName());
+  }
+
+  @Test
+  public void testExportCommand() throws Exception {
+    try (AccumuloClient client = Accumulo.newClient().from(getClientProps()).build()) {
+
+      final String tableName = getUniqueNames(1)[0];
+      client.tableOperations().create(tableName);
+
+      // add splits to table
+      SortedSet<Text> partitions = new TreeSet<>();
+      for (String s : row_numbers) {
+        partitions.add(new Text(s));
+      }
+      client.tableOperations().addSplits(tableName, partitions);
+
+      try (BatchWriter bw = client.createBatchWriter(tableName)) {
+        for (int i = 1; i <= 5000000; i++) {
+          Mutation m = new Mutation(Integer.toString(i));
+          if (i % 2 != 0) {
+            m.put(Integer.toString(i), "", String
+                .format("file://localhost:8020/accumulo/tables/1/default_tablet/I00000%dp.rf", i));
+          } else {
+            m.put(Integer.toString(i), "", String
+                .format("file://localhost:8020/accumulo/tables/2/default_tablet/I00000%dp.rf", i));
+          }
+          bw.addMutation(m);
+        }
+      }
+
+      client.tableOperations().compact(tableName, null, null, true, true);
+      client.tableOperations().flush(tableName, null, null, true);
+
+      Path outputDir = new Path(cluster.getTemporaryPath(), getClass().getName());
+      Path exportDir = new Path(outputDir, "export");
+      client.tableOperations().offline(tableName, true);
+      client.tableOperations().exportTable(tableName, exportDir.toString());
+
+      try (Scanner scanner = client.createScanner(MetadataTable.NAME, Authorizations.EMPTY)) {
+        scanner.setRange(new Range("1", "1<"));
+        scanner.fetchColumnFamily(MetadataSchema.TabletsSection.DataFileColumnFamily.NAME);
+
+        for (Map.Entry<Key,Value> entry : scanner) {
+          log.info("Key is: " + entry.getKey());
+          log.info("Value is: " + entry.getValue());
+          boolean inV1 = entry.getKey().getColumnQualifier().toString().contains(v1.toString());
+          boolean inV2 = entry.getKey().getColumnQualifier().toString().contains(v2.toString());
+          assertTrue(inV1 || inV2);
+        }
+      }
+
+      FileSystem fs = cluster.getFileSystem();
```

Comment:
I would probably be better to move lines 151-153 up to where we define configure v1 and v2.